Documentation/docs-mobile/building-apps/build-properties.md

```diff
@@ -1019,6 +1019,13 @@ specifying the Java package names of generated Java source code.
 The only supported value is
 `LowercaseCrc64`.
 
+## AndroidPrepareForBuildDependsOn
+
+A semi-colon delimited property that can be used to extend the
+Android build process. MSBuild targets added to this property
+will execute early in the build for both Application and Library
+project types. This property is empty by default.
```

Comment:
maybe include a small example of how to extend the property with a custom target?

Reply:
Added an example below